aa-sdk/core/src/actions/bundler/getUserOperationReceipt.ts

```diff
@@ -8,10 +8,11 @@ export const getUserOperationReceipt = async <
   client: TClient,
   args: {
     hash: Hex;
+    tag?: "pending" | "latest";
   },
 ): Promise<UserOperationReceipt | null> => {
   return client.request({
     method: "eth_getUserOperationReceipt",
-    params: [args.hash],
+    params: [args.hash, args.tag],
```

Comment:
just to confirm, is rundler happy w/ this if the params are `[hash, null]`?

Reply:
also will this potentially cause problems w/ [3rd party bundlers](https://www.alchemy.com/docs/wallets/third-party/bundlers), or is that not a concern?